pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```diff
@@ -110,6 +110,8 @@ qtModule {
     # it fails when compiled with -march=sandybridge https://github.com/NixOS/nixpkgs/pull/59148#discussion_r276696940
     # TODO: investigate and fix properly
     "-march=westmere"
+  ] ++ lib.optionals stdenv.isClang [
```

Comment:
```suggestion
  ] ++ lib.optionals stdenv.cc.isClang [
```

```
error: attribute 'isClang' missing

       at /home/jon/.cache/nixpkgs-review/pr-126411/nixpkgs/pkgs/development/libraries/qt-5/modules/qtwebengine.nix:113:22:

          112|     "-march=westmere"
          113|   ] ++ lib.optionals stdenv.isClang [
             |                      ^
          114|     "-Wno-elaborated-enum-base"

       … while evaluating 'optionals'
```

Reply:
Good catch. Should definitely have checked this.